src/test/resources/fixtures/hmh/hmh-rce/development/caliperEventNavigationEvent.json

```diff
@@ -0,0 +1,75 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:c942788b-aae8-4a15-8135-b5ec731a0fc3",
+  "type": "NavigationEvent",
+  "actor": {
+    "id": "urn:uuid:ff7c53fd-9024-4d2d-8503-aa3da2cbc73e",
+    "type": "Person"
+  },
+  "action": "NavigatedTo",
+  "object": {
+    "id": "http://rce.localhost/content/book/OPS/page0001.xhtml",
+    "type": "Frame",
+    "name": "Cover Page",
+    "dateCreated": "2019-06-05T14:17:34.739Z",
+    "extensions": {
+      "contentRootId": "9780358087762",
+      "contentRootType": "pubId",
+      "contentVersionId": 0,
+      "userRole": "student",
+      "contentItemId": "xhtml_page0001"
+    },
+    "isPartOf": {
+      "id": "http://rce.localhost/digitalresource/9780358087762",
+      "type": "DigitalResource",
+      "name": "Hero Academy 2019",
+      "isPartOf": {
+        "id": "NoProgrammeId",
+        "type": "DigitalResource",
+        "name": "NoProgramme"
+      },
+      "version": "1.0"
+    },
+    "index": 1
+  },
+  "eventTime": 1559744254740,
```

Comment:
This eventTime should be ISO8601 format. The spec says `An ISO 8601 date and time value expressed with millisecond precision that indicates when the Event occurred. The value MUST be expressed using the format YYYY-MM-DDTHH:mm:ss.SSSZ set to UTC with no offset specified.`

Reply:
Cool